ran-core/src/main/java/io/ran/Property.java

```diff
@@ -149,7 +203,7 @@ public KeySets keys() {
 				}
 			}
 			if (keys.isEmpty() && contains(Token.of("id"))) {
```

Comment:
this means that if no key annotation was added to the class, will try to retrieve as a key a field called id right?